client/src/hooks/queries/event/useRequestPostEvent.ts

```diff
@@ -1,10 +1,23 @@
-import {useMutation} from '@tanstack/react-query';
+import {useMutation, useQueryClient} from '@tanstack/react-query';
 
 import {RequestPostEvent, requestPostEvent} from '@apis/request/event';
 
+import QUERY_KEYS from '@constants/queryKeys';
+
 const useRequestPostEvent = () => {
+  const queryClient = useQueryClient();
+
   const {mutate, mutateAsync, ...rest} = useMutation({
     mutationFn: ({eventName, password}: RequestPostEvent) => requestPostEvent({eventName, password}),
+    onSuccess: () => {
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.steps]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.event]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.allMembers]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.currentMembers]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.reports]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.billDetails]});
+      queryClient.invalidateQueries({queryKey: [QUERY_KEYS.images]});
+    },
   });
```

Comment:
```suggestion
    onSuccess: () => {
      queryClient.removeQueries();
   });
```

이벤트가 생성될 때 모든 캐싱 데이터를 날리고 싶다면 위와 같이 선언해도 됩니다!

Reply:
리쿼장쿠. 리액트 쿼리 장인 쿠키

Reply:
해당 리뷰의 반영은 #711 에서 반영해둘게요! 토다리가 하지 않아도 괜찮습니댱

Reply:
감사해용~!